.dockerignore

```diff
@@ -2,3 +2,9 @@ data
 logs
 .env
 dist
+node_modules/
```

Comment:
Don't know how I have missed adding node_modules to `.ingore` files 🤦🏻

Reply:
In my projects i use collection of those ignore files:
https://github.com/github/gitignore
Maybe it will be useful in future